packages/base/package.json

```diff
@@ -25,17 +25,17 @@
     "@metamask/auto-changelog": "^3.0.0",
     "eslint": "^8.27.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsdoc": "^41.1.2",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-promise": "^6.1.1",
     "prettier": "^2.7.1"
   },
   "peerDependencies": {
     "eslint": "^8.27.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsdoc": "^39.6.2 || ^41 || ^43.0.7",
```

Comment:
The versions referenced in each README should be updated as well

Reply:
What do we recommend that people install for new projects? Should it be `^43.0.7`?

Reply:
Good question. I guess that could stay the same for now, though the `eslint-plugin-import` reference should be updated at least.